src/com/jester/view/View.java

```diff
@@ -169,18 +203,44 @@ public void valueChanged(ListSelectionEvent event) {
             @Override
             public void actionPerformed(ActionEvent e) {
                 try {
+                    // @TODO - Add newly inserted records here
                     //Insert the code here to save the changes and add the music to
                     //your music library.
                     Thread.sleep(250);
                     JOptionPane.showMessageDialog(null, "Changes Saved", "Confirmation", JOptionPane.PLAIN_MESSAGE);
                 } catch (InterruptedException ex) {
                     Logger.getLogger(View.class.getName()).log(Level.SEVERE, null, ex);
                 }
-                musicTextField.setText(null);
-                artistTextField.setText(null);
-                albumTextField.setText(null);
-                pathTextField.setText(null);
-                lyricsTextField.setText(null);
+                
+                // Start generating unique ID
+                music = new Music();  
+                // Placeholders for attributes in music
+                String new_music_title = musicTextField.getText(); 
+                String new_music_artist = artistTextField.getText(); 
+                String new_music_album = albumTextField.getText(); 
+                String new_music_path = pathTextField.getText(); 
+                String new_lyric_path = lyricsTextField.getText();
+                
+                System.out.println("Saved Changes!");
+                System.out.println("=============================");
+                System.out.println("Music title: " + new_music_title);
+                System.out.println("Music artist: " + new_music_artist);
+                System.out.println("Music album: " + new_music_album);
+                System.out.println("Path file: " + new_music_path);
+                System.out.println("Lyric file: " + new_lyric_path);
+                System.out.println("=============================");
+                
+                // Set up and assign the rest of the attributes
+                music.setMusic_title(new_music_title);
+                music.setMusic_artist(new_music_artist);
+                music.setMusic_album(new_music_album);
+                music.setMusic_path_to_DIR(new_music_path);
+                // Final instantiate to attributes
+                music = new Music(music.getMusic_ID(), new_music_title, new_music_artist, new_music_album, new_music_path);
+                // Call the controller to insert the records
+                controller.SetNewMusic(music);

```

Comment:
Inserts the new music records into the database. 
Includes all Music class members